clonelists/Nintendo - Wii U (Redump).json

```diff
@@ -77,6 +84,13 @@
 				{"searchTerm": "FIFA Soccer 13"}
 			]
 		},
+		{
+			"group": "Fist of the North Star - Ken's Rage 2",
```

Comment:
`Fist of the North Star - Ken's Rage 2` isn't in the Redump DAT, so should be omitted. Titles aren't added speculatively, otherwise they create noise when testing.

Reply:
You're totally right!
I was mixing the redump dat and some eshop games.
My bad!

Reply:
I've fixed this.